src/lib/features/feature-search/feature-search-store.ts

```diff
@@ -245,6 +248,8 @@ class FeatureSearchStore implements IFeatureSearchStore {
                         'features.created_by_user_id',
                     );
 
+                this.findFeatureStrategies(query);
```

Comment:
This method returns after applying additional query operations on the query, but the return appears to be discarded? Are they saved in the query object passed into the method or should we actually be doing a `query = this.findFeatureStrategies(query);` here?

Reply:
Yes, removed return. It will be saved.